lib/importers/google/satisfaction/call_record.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+module Importers
+  module Google
+    module Satisfaction
+      class CallRecord
+        SATISFACTION_VALUE = {
+          'Delighted' => 4,
+          'Vey pleased' => 3,
+          'Very pleased' => 3,
+          'Satisfied' => 2,
+          'Frustrated' => 1,
+          'Very frustrated' => 0
+        }.freeze
+
+        def initialize(cells, row_index)
+          @cells = cells
+          @row_index = row_index
+        end
+
+        def params
+          {
+            uid: uid,
+            given_at: given_at,
+            satisfaction_raw: satisfaction_raw,
+            satisfaction: satisfaction,
+            location: location,
+            delivery_partner: self.class::DELIVERY_PARTNER
```

Comment:
Is there any reason these constants can't instead be readers that are overridden in the derived classes? It would remove the need for the `self.class::` scope busting shenanigans and feels slightly nicer design-wise.